### ISSUE TITLE
für dev: 

### ISSUE THREAD

**Issue by uppzz** [coders4help/volunteer_planner#468]
**Describe the bug**
If a layer has 2 slots and both layers are already occupied and a device changes this already applied layer to one slot,
none of the 2 logged in gets a message and both are still logged in although only one slot exists.

Wenn eine Schicht 2 Slots hat und schon beide Schichten sind belegt und eine Einrichtung ändert diese schon angewendete Schicht auf einen Slot ab,
bekommt keiner der 2 angemeldeten eine Nachricht und beide sind weiterhin angemeldet obwohl nur noch ein Slot existiert.

![image](https://user-images.githubusercontent.com/46990472/159098058-195a677e-2760-42cb-a46e-646390a03bf8.png)
![image](https://user-images.githubusercontent.com/46990472/159098077-d6cf3261-4204-4439-bb49-3ef28071d777.png)

**To Reproduce**

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by christophmeissner** [comment]
What is the expected behaviour?

**Comment by uppzz** [comment]
yes i think that is a good solution that the manager has to take care of it. but can the manager even take an already registered helper out of a shift? well he could write to the last helpers and cancel them manually would need then but a request from the system

**Comment by cyroxx** [comment]
Downsizing a shift is not an edge case, but a totally valid case for shift management.

As a shift manager, when downsizing a shift, I want the system to detect a then-overbooked shift and suggest a list of helpers that will be cancelled (the ones that enrolled last). I can then either just confirm the suggested list or make changes to it (choose the required number of cancellations among all enrolled shift helpers, suggested list is pre-selected). The helpers of the cancelled shifts are notified.
